Mathlib/Algebra/Order/AbsoluteValue.lean

```diff
@@ -301,23 +301,31 @@ variable {S : Type _} [OrderedSemiring S]
 
 variable {R : Type _} [Semiring R] (abv : R → S) [IsAbsoluteValue abv]
 
+lemma abv_nonneg (x) : 0 ≤ abv x := abv_nonneg' x
+
+lemma abv_eq_zero {x} : abv x = 0 ↔ x = 0 := abv_eq_zero'
+
+lemma abv_add (x y) : abv (x + y) ≤ abv x + abv y := abv_add' x y
+
+lemma abv_mul (x y) : abv (x * y) = abv x * abv y := abv_mul' x y
```

Comment:
Should these get #align statements?

Reply:
They should align automatically, but I've added the #align's nevertheless.